_episodes_rmd/08-plot-ggplot2.Rmd

```diff
@@ -278,7 +281,11 @@ variables and their visual representation.
 > > ## Solution to challenge 4b
 > >
 > > Here a possible solution: 
-> >
+> > Notice that supplying the color argument inside the aes() functions enables you to 
```

Comment:
Add code formatting to `aes()` and `color` (and remove single quotes from the latter).